docs/source/getting-started/app-architecture.rst

```diff
@@ -156,14 +156,14 @@ The wrapper function ``follow`` is then passed to the subcomponents of ``MainVie
 For example, ``follow`` is passed to the ``UserList`` component as an argument (a `prop <https://reactjs.org/docs/components-and-props.html>`_ in React terms).
 This gets triggered when you click the icon next to a user's name in the *Network* panel.
 
-.. literalinclude:: code/ui-before/MainView.tsx
+.. literalinclude:: code/templates-tarball/create-daml-app/ui/src/components/MainView.tsx
   :language: tsx
-  :start-after: // USERLIST_BEGIN
-  :end-before: // USERLIST_END
+  :start-after: {/* USERLIST_BEGIN */}
```

Comment:
Why has this changed?

Reply:
The `//` style of comment doesn't work within a JSX block. This code wasn't being compiled before so it didn't matter, now it's a proper comment in the JSX.

Reply:
That makes a lot of sense. Thanks for the explanation.